annotator-core/src/main/java/edu/ucr/cs/riple/core/adapters/NullAwayVersionAdapter.java

```diff
@@ -55,7 +56,7 @@ public interface NullAwayVersionAdapter {
    * @param values Values in row of a TSV file.
    * @return Corresponding Error instance with the passed values.
    */
-  Error deserializeError(String[] values);
+  Error deserializeError(String[] values, FieldDeclarationStore store);
```

Comment:
Can we add the new parameter to the Javadoc?

Reply:
Sure, 0dac9d497fff0365b24a3b004094d51ea720ab81